cluster/gce/config-default.sh

```diff
@@ -35,6 +35,7 @@ REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-true}
 PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 PREEMPTIBLE_MASTER=${PREEMPTIBLE_MASTER:-false}
 KUBE_DELETE_NODES=${KUBE_DELETE_NODES:-true}
+KUBE_DELETE_NETWORK=${KUBE_DELETE_NETWORK:-true}
```

Comment:
I could leave these as `false` for the default case. I'm not sure if there was a strong reason for leaking the firewalls/networks by default besides laziness (@thockin's word, not mine. :)